src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Emitter/CoreBindingHelpers.cs

```diff
@@ -409,7 +409,7 @@ void EmitBindImplForMember(MemberSpec member)
                         member,
                         member.Name,
                         sectionPathExpr: GetSectionPathFromConfigurationExpression(configKeyName),
-                        canSet: true,
+                        canSet: true, canGet: true,
```

Comment:
>canGet: true, [](http://example.com/codeflow?start=38&length=13)

How you guarantee the property has public `get`?

Reply:
This is part of [`EmitInitializeMember`](https://github.com/dotnet/runtime/blob/3393b4d564660e76178c442766f92158f04cfea2/src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Emitter/CoreBindingHelpers.cs#L320) which is used to generate code that 
gets the parameter default, store it to a local variable, then calls a constructor.  It does not refer to a property.  For example:
 https://github.com/dotnet/runtime/blob/3393b4d564660e76178c442766f92158f04cfea2/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/netcoreapp/DefaultConstructorParameters.generated.txt#L81-L85

Since this is working with a local, it may always be get and set freely.  It does not cause different code-gen, though, [since `defaultValueSource: null`](https://github.com/dotnet/runtime/pull/102733/files/2eb090df285e32e927f7c856e18336610419b8c6#diff-91b1e1d53f7b0463c9b65f5f45c3dfe220e8249d07b1c9168cdff4e6eac82c3bR701).